addons/account/tests/test_account_invoice_report.py

```diff
@@ -20,10 +20,11 @@ def setUpClass(cls):
                 'invoice_date': fields.Date.from_string('2016-01-01'),
                 'currency_id': cls.other_currency.id,
                 'invoice_line_ids': [
-                    (0, None, {
+                    Command.create({
                         'product_id': cls.product_a.id,
                         'quantity': 3,
-                        'price_unit': 750,
+                        'price_unit': 4500,
+                        'product_uom_id': 2,    # pack of 6
```

Comment:
Never hardcode an id directly, even if we do not test with demo data that id could change in the future